docker/build_windows.dockerfile

```
@@ -15,7 +15,9 @@ RUN git lfs install
 RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
       ttf-mscorefonts-installer \
-      wine-stable
+      wine-stable \
+      make \
+      aria2
```

```suggestion
      wget
```

I think it's better to use `wget` for this. Using wget, you can also retry the download:

https://superuser.com/questions/493640/how-to-retry-connections-with-wget

---

Ok @benjaoming, thanks 👍. I will change that.